samples/ipc/ipc_service/sample.yaml

```diff
@@ -1,52 +1,120 @@
 sample:
-    name: IPC Service integration test
-    description: IPC Service integration and efficiency test
+  name: IPC Service integration test
+  description: IPC Service integration and efficiency test
+
+common:
+  build_only: true
+  sysbuild: true
+  tags: ipc
+
 tests:
-  sample.ipc.ipc_service.rpmsg_default:
-    build_only: true
-    platform_allow: nrf5340dk_nrf5340_cpuapp
+  sample.ipc.ipc_service.nrf5340dk_rpmsg_default:
+    platform_allow:
+      - nrf5340dk_nrf5340_cpuapp
     integration_platforms:
       - nrf5340dk_nrf5340_cpuapp
-    tags: ipc
-  sample.ipc.ipc_service.rpmsg_cpuapp_sending:
-    build_only: true
-    platform_allow: nrf5340dk_nrf5340_cpuapp
+    extra_configs:
+      - SB_CONFIG_REMOTE_BOARD='"nrf5340dk_nrf5340_cpunet"'
```

Comment:
`SB_CONFIG_*` should be under `extra_args` to work with twister.
Putting them under `extra_configs` only works with `west build -T` (which in itself is a problem).

Reply:
this actually doesn't work, I get:

```
/home/gmarull/ws/nordic/sdk-nrf/nrf/build/zephyr/misc/generated/extra_kconfig_options.conf:2: warning: malformed string literal in assignment to REMOTE_BOARD (defined at /home/gmarull/ws/nordic/sdk-nrf/nrf/samples/ipc/ipc_service/Kconfig.sysbuild:9). Assignment ignored.
```

it looks like file doesn't quote properly the Kconfig options...
```
SB_CONFIG_REMOTE_BOARD=nrf54h20pdk_nrf54h20_cpurad
```

```
    extra_configs:
      - SB_CONFIG_REMOTE_BOARD='"nrf5340dk_nrf5340_cpunet"'
```

The quote hack above works with `west build -T`.

Reply:
resorted to `SB_CONF_FILE` workaround until quoting issues are fixed upstream.